src/map_io/s2map.cc

```diff
@@ -552,35 +552,26 @@ void S2MapLoader::load_s2mf(Widelands::EditorGameBase& egbase) {
 		throw wexception("Section 5 (Bobs) not found");
 	}
 
-	//  SWD-SECTION 6: Ways
-	//  This describes where you can put ways
-	//  0xc* == it's not possible to build ways here now
-	//  0x80 == Heres a HQ, owner is Player number
-	//   owner == 0 -> blue
-	//   owner == 1 -> yellow
-	//   owner == 2 -> red
-	//   owner == 3 -> pink
-	//   owner == 4 -> grey
-	//   owner == 6 -> green
-	//   owner == 6 -> orange
-	// TODO(hessenfarmer): this comment is not correct section 6 determines how the values of section
-	// 5 are to be interpreted. Solution is working more the less though
-	section = load_s2mf_section(fr, mapwidth, mapheight);
-	if (!section) {
-		throw wexception("Section 6 (Ways) not found");
+	//  SWD-SECTION 6: Object type
+	//  Describes the meaning of section 5
+	std::unique_ptr<uint8_t[]> object_type(load_s2mf_section(fr, mapwidth, mapheight));
+	if (object_type == nullptr) {
+		throw wexception("Section 6 (Object Type) not found");
 	}
 
+#if 0  // NOCOM
```

Comment:
@Noordfrees shouldn't we have deleted this, after testing?

Reply:
Oops, I hadn't meant to commit that, sorry :(

Removed in db05f9a02da524e617685eb84a8717115a0dfe8c